Automattic/jetpack
# Record stat of the first time the site is registered

Under the `jetpack-connections` MC Stat there's an entry for `unique-connection`, that is bumped everytime a site connects for the first time.

This is triggered only after the user authorization. 

This PR adds a new similar metric for `unique-registrations` that is triggered upon site registration and will therefore count sites with a user-less connection.

#### Changes proposed in this Pull Request:

* Adds unique-registrations MC Stats

#### Jetpack product discussion


1191179647901802-as-1199552431277341

#### Does this pull request change what data or activity we track or use?


Yes, we now anonymously track the first time a site was registered under `jetpack-connections/unique-registrations` MC Stat

#### Testing instructions:






* Checkout this branch
* Visit MC Stat `jetpack-connections/unique-registrations` and look at the count
* Make sure this option does not exist: `wp jetpack options get unique_registrations`
* If your site is connected, disconnect
* Click Set Up Jetpack, don't connect the user
* Make sure the option value is `1`: `wp jetpack options get unique_registrations`
* Visit MC Stat `jetpack-connections/unique-registrations` and see the count was bumped
* Refresh the page, you will be asked to Set up again. Set up!
* Make sure the option value is `2`: `wp jetpack options get unique_registrations`
* * Visit MC Stat `jetpack-connections/unique-registrations` and make sure the count was NOT bumped
